deps/v8/src/torque/contextual.h

```diff
@@ -82,12 +82,21 @@ class ContextualVariable {
   struct VarName                                  \
       : v8::internal::torque::ContextualVariable<VarName, __VA_ARGS__> {}
 
+#if defined(V8_OS_ZOS)
```

Comment:
Should we mark this with comment `//FIXME: revert when Clang compiler is available` where thread_local is supported?